### PR TITLE
EMIP Metadata

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
Correction to witness statements in LIT6347 and creation of new work, following GitHub discussion

### CODE REVIEW

File: new/LIT6373CanonAgreement.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6373CanonAgreement" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በእንተ፡ ፹ወ፩መጻሕፍት፡ ከመ፡ ፩ቃል፡ እማንቱ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">bʾǝnta samānǝyā wa ʾaḥadu maṣaḥeft kama ʾaḥadu qal ʾǝmantu</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6373CanonAgreement.xml:9]: I would say: Baʾǝnta .... samānǝyā-wa-ʾaḥadu

Comment (DenisNosnitsin1970) [reply new/LIT6373CanonAgreement.xml:9]: maṣaḥǝft

File: new/LIT6373CanonAgreement.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <title xml:lang="en" corresp="#t1">Agreement of the Eighty-One Canonical Scriptures</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6373CanonAgreement.xml:10]: Perhaps you could get the translation a little closer to the original text.

File: new/LIT6373CanonAgreement.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                <title xml:lang="en" corresp="#t1">Revelation of a miracle according John Son of Thunder which proclaims the Eighty-One Canonical Scriptures</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6373CanonAgreement.xml:11]: Revelation... must be #t2 (title 2)

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
I used to list one or two witnesses in "relation"; I think quotation from the incipit/explicit are necessary.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
For the title of LIT6347SalamWatchful, I would stile the title as: "Greeting to you, the congregation of diligent angels..." as you take the first text line. "Greetings to the congregation..." will be #t2. 
Please check ሰላምል፡ in "incipit"

File: new/LIT6373CanonAgreement.xml (hunk 2 above)
Comment (Ralph-Lee-UK) [reply new/LIT6373CanonAgreement.xml]: I've had a go, let me know what you think

File: new/LIT6373CanonAgreement.xml (hunk 3 above)
Comment (Ralph-Lee-UK) [reply new/LIT6373CanonAgreement.xml]: Done

**Comment by Ralph-Lee-UK** [comment]
@DenisNosnitsin1970 , I take your point on the title. I think I have address this. Happy to revise further if you think it will improve it

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**